ui/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 ### Added
+- NativeSQL translator: Added support for the duration step
```

Comment:
Maybe we should precise Postgres, Athena and BigQuery?
If I understand correctly, the other engines will fail during `_interval_to_seconds` because they don't override it.

Reply:
Ah ye! Thanks for the clarification :+1: